pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/converter/stats/MutableNoDictionaryColStatistics.java

```diff
@@ -125,4 +125,13 @@ public CLPStats getCLPStats() {
     throw new IllegalStateException(
         "CLP stats not available for column: " + _dataSourceMetadata.getFieldSpec().getName());
   }
+
```

Comment:
```
public CLPStats getCLPStats() {
    if (_forwardIndex instanceof CLPMutableForwardIndex) {
      return ((CLPMutableForwardIndex) _forwardIndex).getCLPStats();
    } else if (_forwardIndex instanceof CLPMutableForwardIndexV2) {
      return ((CLPMutableForwardIndexV2) _forwardIndex).getCLPStats();
    }
```
Saw similar code in getCLPStats() function, can you please let me know in which scenarios these functions are being called.

Reply:
ClpMutableForwardIndexV2 index -> ClpForwardIndexCreator (V1): use `getCLPStats()`
ClpMutableForwardIndexV2 index -> ClpForwardIndexCreatorV2: use `getCLPV2Stats()`

`getCLPStats()` method is basically for compatibility with the initial version of immutable forward index.